genet/core.py

```diff
@@ -2200,54 +2202,37 @@ def add_elevation_to_nodes(self, elevation_tif_file_path, null_value: float):
                 z = 0
             elevation_dict[node_id] = {'z': z}
 
-        self.apply_attributes_to_nodes(elevation_dict)
+        if run_validation is True:
+            print(elevation.validation_report_for_node_elevation(elevation_dict))
```

Comment:
Not sure about printing this dictionary. In some (bad) cases I guess this dict can get pretty big (the 'too high' and 'too low' values are reported for each node). 
Instead of print, let's use `logging.info` and report only the summary part of the dict? and if that looks bad, someone can run the validation and get the output as a dict that they can do something with.

Reply:
Yeah that sounds like a good idea!